draft-ietf-trans-rfc6962-bis.md

```diff
@@ -552,14 +552,15 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
 (MMD), after the issuance of the SCT. Periodically, the log MUST append all its
 new entries to its Merkle Tree and sign the root of the tree.
 
-Log operators MUST NOT impose any conditions on retrieving or sharing data from
-the log.
+Log operators SHOULD NOT impose any conditions on retrieving or sharing data
+from the log.
 
 ## Accepting Submissions
 
-Before accepting a submitted certificate or precertificate, the log MUST verify
-that it has a valid signature chain to an accepted trust anchor, using the chain
-of intermediate CA certificates provided by the submitter. Logs SHOULD accept
+To avoid logs being overloaded by invalid submissions, before accepting a
+submitted certificate or precertificate, the log MUST verify that it has a
+valid signature chain to an accepted trust anchor, using the chain of
+intermediate CA certificates provided by the submitter. Logs SHOULD accept
```

Comment:
"logs...the log" mixes singular and plural unnecessarily.

I suggest rewording this sentence as follows:
"To avoid being overloaded by invalid submissions, the log MUST NOT accept any
submission until it has verified that the submitted certificate or
precertificate has a valid signature chain to an accepted trust anchor, using
the chain of intermediate CA certificates provided by the submitter."

What do you think?